keras/models.py

```diff
@@ -577,8 +577,15 @@ def build(self, input_shape=None):
             raise TypeError('Sequential model cannot be built: model is empty.'
                             ' Add some layers first.')
         # actually create the model
-        self.model = Model(self.inputs, self.outputs[0],
-                           name=self.name + '_model')
+        if K.backend() == 'mxnet' and hasattr(self, '_context') and \
+                self._context is not None:
```

Comment:
can use short circuit here - move self._context is not None condition first and follow with the rest

Reply:
It fails if `_context` parameter is not part of an object(self or sequential() here), that's why I am checking first whether `_context` parameter is available as a class variable or not. Based on the different backend, `self._context` parameter is not added in the constructor.

Reply:
Makes sense. Thanks!